packages/tradershub/src/components/StaticLink/StaticLink.tsx

```diff
@@ -17,17 +17,18 @@ type StaticLinkProps = {
  * @param {string} [props.className] - Optional additional CSS classes to apply.
  * @param {AnchorHTMLAttributes<HTMLAnchorElement>['href']} [props.href] - The URL that the link points to.
  * @param {AnchorHTMLAttributes<HTMLAnchorElement>['href']} [props.staticUrl] - A static URL that the link points to.
- *
  * @returns {ElementType} The `StaticLink` component.
  */
 const StaticLink = ({ children, className, href, staticUrl, onClick }: StaticLinkProps) => {
+    const link = href ?? (staticUrl && getStaticUrl(staticUrl));
+    const isNewTab = href || staticUrl;
     return (
         <a
             className={twMerge('underline text-brand-coral py-0 px-4 underline-offset-2', className)}
-            href={href ?? (staticUrl ? getStaticUrl(staticUrl) : '#')}
+            href={link}
             onClick={onClick}
-            rel='noopener noreferrer'
-            target='_blank'
+            rel={isNewTab && 'noopener noreferrer'}
+            target={isNewTab && '_blank'}
```

Comment:
can't we just add this as `newTab` and by default `false`?

Reply:
In this case we need to write one more prop (newTab) for the component, when we pass `href` or `staticUrl`. In my opinion, handling it inside of the component is better

Reply:
I agree @sergei-deriv. Lemme handle external and internal links